envpool/mujoco/dmc/mujoco_dmc_deterministic_test.py

```diff
@@ -116,6 +118,11 @@ def test_pendulum(self) -> None:
     for task in ["swingup"]:
       self.check(DmcPendulumEnvSpec, DmcPendulumDMEnvPool, task, obs_keys)
 
+  def test_point_mass(self) -> None:
+    obs_keys = ["position"]  # "velocity"
```

Comment:
The “velocity” of o1 and o2 are all the same in every step. Do not know if it is a bug or needs to add a white list.

Reply:
whitelist for easy